intelmq/bots/collectors/mail/collector_mail_url.py

```diff
@@ -38,9 +38,12 @@ def process(self):
                 self.logger.info("Reading email report")
 
                 for body in message.body['plain']:
-                    match = re.search(self.parameters.url_regex, body)
+                    match = re.search(self.parameters.url_regex, str(body))
```

Comment:
Why should this not be a string? Could this be a bytes in a special case? Could you describe/show such a case?